<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>153</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>500</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>스밍해</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">글번호</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>닉네임</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>제목</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="smingListTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">시간</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>곡명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>제목</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="imgControl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>res/play1.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputid">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="placeholderText">
     <string>디시 아이디</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputpw">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLatinOnly|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
    <property name="placeholderText">
     <string>디시 비밀번호</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>갤명(lovelyz)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>유동</string>
    </property>
   </widget>
  </widget>
  <action name="action">
   <property name="text">
    <string>설정</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>사용법</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>inputid</tabstop>
  <tabstop>inputpw</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>chk</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>smingListTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slotItemClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imgControl</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_Melon_Play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_dc_login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smingListTable</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sming_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_Get_DC()</slot>
  <slot>btn_Melon_Play()</slot>
  <slot>btn_Melon_Restart()</slot>
  <slot>btn_txt_Enter()</slot>
  <slot>btn_Test()</slot>
  <slot>slotItemClicked()</slot>
  <slot>slot1()</slot>
  <slot>btnTesting()</slot>
  <slot>btn_dc_login()</slot>
  <slot>sming_clicked()</slot>
 </slots>
</ui>
